R/rename.r

```diff
@@ -16,7 +18,16 @@
 #' x
 #' # Rename column "disp" to "displacement"
 #' rename(mtcars, c("disp" = "displacement"))
-rename <- function(x, replace, warn_missing = TRUE) {
+rename <- function(x, replace, warn_missing = TRUE, warn_duplicate = TRUE ) {
+  
+  # This line does the real work of `rename()`.
   names(x) <- revalue(names(x), replace, warn_missing = warn_missing)
```

Comment:
I don't feel strongly --you tell me.  

I believe there are many times where it's valid for `revalue()` to produce duplicates.  However, I think it's almost always a programming mistake for `rename()` to produce duplicates.  I don't remember coming across a scenario where it's desirable to have a dataset with two identical variable names.

Would you like me to move it to `revalue()` so the two functions have a consistent signature (even if it's rarely used in `revalue()`)?  I'll put it where you think it's best.

Since I'm already shaming my mother and compromising my integrating by conforming to your spacing and naming conventions....moving it to `revalue()` won't be so hard.